native/cocos/renderer/gfx-base/GFXDescriptorSet.h

```diff
@@ -78,8 +78,9 @@ class CC_DLL DescriptorSet : public GFXObject, public RefCounted {
     ccstd::vector<ObjectWithId<Buffer>> _buffers;
     ccstd::vector<ObjectWithId<Texture>> _textures;
     ccstd::vector<ObjectWithId<Sampler>> _samplers;
-
     bool _isDirty = false;
+    uint8_t _rsv[3] = {0};
```

Comment:
What does rsv mean? We might use the full name.

Reply:
I should use _padding.

Reply:
I think we don't need to do padding manually. We can assume developers understand c++ object alignment rules.
[c++ object](https://en.cppreference.com/w/cpp/language/object)
If we want to make sure object layout should meet certain requirements, we can use static_assert to check the layout.
For example, `static_assert(sizeof(DescriptorSet) == xxx);` etc.